### DISCUSSION TITLE
无法添加YOUTUBE资源，切换全局代理或更新代理设置都显示错误，请问是否有解决方法？另，AI使用有限额是否有解决办法？

### DISCUSSION THREAD

**Discussion by YuChen-Ethan** [ZuodaoTech/everyone-can-use-english#1113] [Q&A]
![image](https://github.com/user-attachments/assets/132dba0c-ebc1-4631-8c4d-1acf2f895869)
![image](https://github.com/user-attachments/assets/2e534c9e-4e82-43e1-a156-50909ad57222)

**Comment by an-lee** [comment]
代理设置填错了，前面 `export https proxy=` 不需要，就填代理地址就行。

充值就没有限额了。

**Reply by YuChen-Ethan** [reply]
谢谢老师，已解决……

**Comment by PyCodeGuru** [comment]
![image](https://github.com/user-attachments/assets/ca4b361f-d1f8-4390-b913-6f97b1132337)

下载视频时报错了，请问老师这种问题要怎么解决呢？
can't bypass age restriction: login required to confirm your age

**Reply by an-lee** [reply]
这个问题暂时解决不了，就是这条视频有年龄限制，不允许未登录的用户访问，所以没法下载。
可以去找别的 YouTuBe 下载服务，自己下载好之后再导入进来。

**Comment by jayden20166** [comment]
是不能在线资源复制链接下载吗？我复制Youtube相关视频的链接去下载 一直下载不了。是不是只能先下载到本地再导入这样子

**Reply by jiuwenyu** [reply]
下载再倒入限制视频大小，不方便

**Comment by jiuwenyu** [comment]
![微信图片_20250108181247](https://github.com/user-attachments/assets/72150d9f-df38-4e11-851a-cc55d4a9b515)
![微信图片_20250108181255](https://github.com/user-attachments/assets/15815f0f-1d06-429f-9af5-275baed66e85)
这么设置代理，下载youtube视频还是报错，大家有碰到这种问题吗，求解决方法

**Reply by yang-shi-dev** [reply]
不要只填7890，完整的是：http://127.0.0.1:7890

**Reply by jiuwenyu** [reply]
还是不行，奇怪
![image](https://github.com/user-attachments/assets/054575d6-653a-4bab-9177-5d9f485bd65c)

**Reply by jiuwenyu** [reply]
![image](https://github.com/user-attachments/assets/880dd20b-a4a9-459d-83fe-cafab89b054a)

**Reply by ricklei2777** [reply]
还是使用

https://cobalt.tools/

网站下载之后导入好了
